### PR TITLE
Refactor: Make VichImageType class extensible by removing 'final'

### PR DESCRIPTION
**Pull request by Erdou**
This commit updates the VichImageType class by removing the 'final' keyword, thereby allowing for extension of the class. This modification maintains consistency with Symfony's practice of non-final form types, enabling developers to extend this form type to add custom functionality or modifications as needed.

### CODE REVIEW

**Review by garak — CHANGES_REQUESTED** [review]
Could we add the "accept" option as well?

**Comment by Erdou** [comment]
@garak I'm not sure about what you are referring to. Can you precise? :)

**Comment by garak** [comment]
The "attr" option in this comment https://github.com/dustin10/VichUploaderBundle/issues/625#issuecomment-1793430910

**Comment by Erdou** [comment]
@garak Done 👍

File: src/Form/Type/VichImageType.php [hunk 1]
```diff
@@ -43,6 +43,9 @@ public function configureOptions(OptionsResolver $resolver): void
             'image_uri' => true,
             'imagine_pattern' => null,
             'storage_resolve_method' => self::STORAGE_RESOLVE_URI,
+            'attr' => [
+                'accept' => 'image/png,image/jpeg,image/gif,image/webp',
```
Comment (garak) [inline src/Form/Type/VichImageType.php:47]: `'image/*'` is probably better (more synthetic and open to possible other formats)

**Review by garak — CHANGES_REQUESTED** [review]

File: src/Form/Type/VichImageType.php (hunk 1 above)
Comment (Erdou) [reply src/Form/Type/VichImageType.php]: > `'image/*'` is probably better (more synthetic and open to possible other formats)

Agreed, and done

**Review by garak — APPROVED** [review]

**Pull request merged**